app/Http/Controllers/TemplateController.php

```diff
@@ -74,9 +79,33 @@ public function edit($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function update(Request $request, Template $template)
     {
-        //
+        $data = $request->only([
+            'name',
+            'content',
+            'params',
+            'status',
+        ]);
+        $data['user_id'] = Auth::id();
+
+        DB::beginTransaction();
+        try {
+
+            $template = $this->templateRepository->update($template->id, $data);
+
+            DB::commit();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/TemplateController.php</br>**Line**: 97</br>Blank line found at start of control structure